zeth_contracts/contracts/Groth16Mixer.sol

```diff
@@ -41,35 +41,42 @@ contract Groth16Mixer is BaseMixer {
 
     VerifyingKey verifyKey;
 
-    // Constructor
+    // Constructor.  Form of vk is:
+    //    uint256[2] Alpha,
+    //    uint256[4] Beta,
+    //    uint256[4] Delta,
+    //    uint256[] ABC_coords
```

Comment:
Side note: Now that we support several curves (i.e. bn/bls) may be worth to clarify these comments. This contract is tailored for bn254, and won't work with any other groups. So May be worth specifying under with circumstances your comment above holds (i.e. `ALT_BN128` set in cpp  code config) or simply rephrase to make this generic, e.g.:
```
// Constructor. The form of vk is:
//     alpha = vk[:2*N]
//     beta = vk[2*N:6*N]
//     delta = vk[6*N:10*N]
//     ABC_coords = vk[10*N:]
//  where:
//     - q is the characteristic of the base field of the pairing group used
//     - N = 2*\ceil{bitLen(q)/256} (# EVM words necessary to represent elements of Fq)
```

Reply:
Side note2: Making the constructor below generic shouldn't be really hard, but troubles may start with functions below, so that may be better to have different contracts to avoid any complexity related to generalization (and so my comment above may not apply anymore if the group name appears in the contract name)